eradiate/radprops/rad_profile.py

```diff
@@ -484,6 +484,18 @@ class US76ApproxRadProfile(RadProfile):
         type="array",
     )
 
+    absorption_switch = documented(
```

Comment:
This variable name does not reflect to the object's behaviour or properties. I'd rather go with `has_absorption` (see below for an example where an appropriate name makes a difference).

Reply:
I agree, this is much better naming, thank you!